From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 15 Sep 2014 10:52:32 +0200
Subject: net/cert: increase default key length

Should we add something for chrome://settings, or just leave
it as hidden as it was before?
---
 net/cert/x509_util.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/cert/x509_util.cc b/net/cert/x509_util.cc
--- a/net/cert/x509_util.cc
+++ b/net/cert/x509_util.cc
@@ -243,7 +243,7 @@ bool GetTLSServerEndPointChannelBinding(const X509Certificate& certificate,
 }
 
 // RSA keys created by CreateKeyAndSelfSignedCert will be of this length.
-static const uint16_t kRSAKeyLength = 1024;
+static const uint16_t kRSAKeyLength = 2048;
 
 // Certificates made by CreateKeyAndSelfSignedCert will be signed using this
 // digest algorithm.
--
2.25.1
